lib/Makefile.mk

```diff
@@ -324,7 +324,7 @@ ifdef WIN32
   _LIBS += -lws2_32 -lcrypt32 -lbcrypt
 endif
 
-ifneq ($(findstring 11,$(subst $() ,,$(SSLLIBS))),)
+ifneq ($(findstring 11,$(subst $(subst ,, ),,$(SSLLIBS))),)
```

Comment:
Would something like $(space) work

Reply:
I'm not aware of such built-in function. We can initialize such variable using the same hack, but I'd only do it once we have another place which needs a space at a problematic location.

Reply:
Ok. It would be easier to understand IMO but that's not critical.